docs/topics/i18n/timezones.txt

```diff
@@ -26,8 +26,11 @@ to this problem is to use UTC in the code and use local time only when
 interacting with end users.
 
 Time zone support is disabled by default. To enable it, set :setting:`USE_TZ =
-True <USE_TZ>` in your settings file. Time zone support uses pytz_, which is
-installed when you install Django.
+True <USE_TZ>` in your settings file. By default, time zone support uses pytz_,
+which is installed when you install Django; Django also supports the use of
+other time zone implementations like zoneinfo_ by passing
```

Comment:
I think you can do without hard coding the link by using
```suggestion
other time zone implementations like :mod:`zoneinfo` by passing
```
(If you do this, just don't forget to delete the `.. _zoneinfo:` line at the end)

Reply:
Ah yeah, originally I wasn't sure if this sometimes renders with references to the Python < 3.8 documentation (in which case it would be a dead link), but considering I use a `:func:` reference elsewhere I think it's a safe bet to go with this.

Reply:
Renders with current Python, i.e., >= 3.9:

https://github.com/django/django/blob/e0a46367df8b17905f1c78f5c86f88d21c0f2b4d/docs/conf.py#L148-L149